h5grove/tornadoutils.py

```diff
@@ -33,27 +33,24 @@ def get(self):
             raise MissingArgumentError("file")
 
         path = self.get_query_argument("path", None, strip=False)
-        format_arg = self.get_query_argument("format", None)
 
         full_file_path = os.path.join(self.base_dir, file_path)
         if not os.path.isfile(full_file_path):
             raise HTTPError(status_code=404, reason="File not found!")
 
         with h5py.File(full_file_path, "r") as h5file:
             try:
-                content = self.get_content(h5file, path)
+                response = self.get_response(h5file, path)
             except NotFoundError as e:
                 raise HTTPError(status_code=404, reason=str(e))
 
-        response = encode(content, format_arg)
-
         for key, value in response.headers.items():
             self.set_header(key, value)
 
         self.write(response.content)
         self.finish()
 
-    def get_content(self, h5file: h5py.File, path: Optional[str]):
+    def get_response(self, h5file: h5py.File, path: Optional[str]) -> Response:
```

Comment:
Specific part returns a h5grove `Reponse`, this allows to change the encoding only for `/data/`